test/e2e/fixture-builder.js

```diff
@@ -1321,15 +1397,6 @@ class FixtureBuilder {
     });
   }
 
-  withNetworkSupportEIP1559() {
```

Comment:
Renamed `withNetworkSupportEIP1559` to `withNetworkControllerSupportEIP1559` so its easy to find, and sorted the file alphabetically